test/es-module/test-esm-tla-unfinished.mjs

```diff
@@ -72,25 +72,27 @@ describe('ESM: unsettled and rejected promises', { concurrency: !process.env.TES
   });
 
   it('should exit for an unsettled TLA promise with warning', async () => {
-    const { code, stderr, stdout } = await spawnPromisified(execPath, [
+    const { code, stdout, stderr } = await spawnPromisified(execPath, [
       fixtures.path('es-modules/tla/unresolved.mjs'),
     ]);
 
-    assert.match(stderr, /Warning: Detected unsettled top-level await at.+unresolved\.mjs:1/);
+    assert.strictEqual(stdout, 'the exit listener received code: 13\n');
```

Comment:
In general, I find it more useful to test stderr first, as it’s more likely to contain useful debugging info if the test fails. Could you change the order please?

Reply:
addressed :+1: